### ISSUE TITLE
proposal: Support multiple packages (and directories) as targets

### ISSUE THREAD

**Issue by haya14busa** [golang/lint#270]
I'm proposing that golint should supports multiple packages as lint targets.

### Rationale

These days lots of packages use vendor feature and when we want to exlude vendor packages from `go test`,
we can do this as follow.

```
go test $(go list ./... | grep -v /vendor/)
```
ref: https://github.com/golang/go/issues/11659#issuecomment-171678025

Also, when we want to run go vet, we can run like this in the same way.

```
go vet $(go list ./... | grep -v /vendor/)
```

However, golint doesn't support multiple packages as lint targets, so we cannot do this.

```
golint $(go list ./... | grep -v /vendor/)
```

e.g.

```
$ golint $(go list ./... | grep -v /vendor/)
open github.com/golang/lint: no such file or directory
open github.com/golang/lint/golint: no such file or directory
```

It's really incovenient. golint supports `golint ./...` but we cannot exlude vendor packages.
`golint directory/...` doesn't help as well.

What do you think?

**Comment by haya14busa** [comment]
I also opens pull-request #271 for implementation.

**Comment by haya14busa** [comment]
fixed by #271
